### ISSUE TITLE
[Request]: update win32 dependency

### ISSUE THREAD

**Issue by jonbhanson** [fluttercommunity/plus_plugins#1775]
### Plugin

share_plus

### Use case

`share_plus` is conflicting with other packages due to its dependency on an old version of the `win32` package:

```
Because share_plus >=6.0.0 depends on win32 ^3.0.0 and file_picker >=5.2.11 depends on win32 ^4.1.3, share_plus >=6.0.0 is incompatible with file_picker >=5.2.11.
So, because your app depends on both file_picker ^5.2.11 and share_plus ^6.3.4, version solving failed.
```

### Proposal

Please update the `win32` dependency to the latest version.

**Comment by vbuberen** [comment]
This is already done in one of recently merged PRs. Please wait for another release of plugins or use `dependency_override`

**Comment by jonbhanson** [comment]
Great, thanks!
